app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -106,11 +107,13 @@ def storage_classes_for_select
       storage_classes.map { |sc| OpenStruct.new(id: sc.name, description: "#{sc.name} (#{sc.provisioner})") }
     end
 
-    def new_volume(attr = {})
-      return unless new_volume_errors.empty?
 
-      vol = Fog::Kubevirt::Compute::Volume.new(attr)
-      vol.boot_order = 1 if attr[:bootable] == "on" || attr[:bootable] == "true"
+    def new_volume(attrs = {})
+      capacity = attrs.delete(:capacity) || "1"
```

Comment:
Why do you set a default ? it is mandatory argument.

Reply:
yes, if the capacity is not mentioned, the creation of the volume will fail.